.github/workflows/scripts/linux/docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+
+branch=`echo ${GITHUB_REF##*/}`
+
+cd /pcsx2
+realpath . && ls -al .
+
+mkdir build && cd build || exit 1
+
+cmake ..                                        \
+    -DCMAKE_C_COMPILER=/usr/lib/ccache/gcc      \
+    -DCMAKE_CXX_COMPILER=/usr/lib/ccache/g++    \
+    -DCMAKE_BUILD_TYPE=Release                  \
+    -DPACKAGE_MODE=TRUE                         \
+    -DXDG_STD=TRUE                              \
+    -DDISABLE_ADVANCE_SIMD=TRUE                 \
+    -DCMAKE_INSTALL_LIBDIR="/tmp/"              \
+    -DCMAKE_INSTALL_DATADIR="/tmp/"             \
+    -G Ninja
+    
+    #-DGTK3_API=TRUE                            \
```

Comment:
any particular reason we aren't using GTK3?  If there's a good reason, might be a good idea to add an explanation here so we switch when the blocker is gone

Reply:
Gtk3 is default, using the cmake flag causes a warning. I was using GTK2, before it broke with a GSdx merge. But kept in the flag as a comment.